differ/content_test.go

```diff
@@ -0,0 +1,65 @@
+package differ
+
+import (
+	"bytes"
+	"encoding/gob"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/RedHatInsights/ccx-notification-service/conf"
+	"github.com/RedHatInsights/ccx-notification-service/types"
+	"github.com/RedHatInsights/insights-content-service/content"
+	"github.com/stretchr/testify/assert"
+)
+
+func makeFetchRequest(testFileName string) (err error, rules types.RulesMap) {
+	config, err := conf.LoadConfiguration(configFileEnvVariableName, defaultConfigFileName)
+	if err == nil {
+		s := httptest.NewServer(http.HandlerFunc(Handler))
+		depencenciesConfiguration := conf.GetDependenciesConfiguration(&config)
+		depencenciesConfiguration.ContentServiceServer = s.URL
+		depencenciesConfiguration.ContentServiceEndpoint = "/tests/content/" + testFileName
+		rules, err = fetchAllRulesContent(&depencenciesConfiguration)
+	}
+	return err, rules
+}
+func TestFetchAllRulesContentOk(t *testing.T) {
+	err, rules := makeFetchRequest("ok")
+	assert.NoError(t, err)
+	allContentFromMap := getAllContentFromMap(rules)
+	assert.NotEmpty(t, allContentFromMap)
+	assert.Len(t, allContentFromMap, 2)
```

Comment:
I guess I myself will update tests to check the returned content more thoroughly later ;)